src/CreateDeckPage.jsx

```diff
@@ -1,14 +1,229 @@
+    updatedDecks.sort((a, b) => {
+      const nameA = a.name.toLowerCase();
+      const nameB = b.name.toLowerCase();
+      return sortType === "A-Z" ? nameA.localeCompare(nameB) : nameB.localeCompare(nameA);
+    });
+    setDecks(updatedDecks);
+  };
+  
+  //Renders list of decks on page
+  function DeckList(props){
+    return <div className="list-group" id="list-tab" key="list">
+      {props.decks.map((deck, index) =>
+        <div className="row" key={index}>
+          <div className="col-10">
+            <button type="button" className="list-group-item list-group-item-action" onClick={()=>{showCardEditor(index)}}>
+              {deck.name}
+            </button>
+          </div>
+          <div className="col-2">
+            <button type="button" className="list-group-item list-group-item-action text-center" onClick={()=>{deleteDeck(index)}}>
+                  Delete
+            </button>
+          </div>
+        </div>)
+      }
+    </div>
+  }
+
+  function showCardEditor(deckIndex){
+    setHiddenDeck(true);
+    setDeckName(decks[deckIndex].name)
+    setDeckIndex(deckIndex)   
+    setCardIndex(0);
+    setAnswerHook(decks[deckIndex].cards[cardIndex].answer);
+    setQuestionHook(decks[deckIndex].cards[cardIndex].question);
+    setCardName(decks[deckIndex].cards[cardIndex].name);
+  }
+
+  function showDecks(){
+    setHiddenDeck(false);
+  }
+    
+  //renders list of cards and shows the individual card when clicked
+  function ListCards(){
+    return <>
+      <label htmlFor="cards">Pick A Card:</label>
+      <select className="form-select" id="cards" size="18">
+        {// ? is if there are no cards
+          decks[deckIndex]?.cards.map((_,index)=>
+            <option className={(index===cardIndex)?"text-primary font-wieght-bold":""}key={index} selected={(index===cardIndex)?true:false} onClick={() => {setCardIndex(index);}}>{"Card "+[index+1]+": "+decks[deckIndex].cards[index].name}</option>)
+        }
+      </select>
+    </>
+  }
+  
+  //save to local storage
+  function saveDecks(){
+    const deckString = JSON.stringify(decks);
+    localStorage.setItem("userDeck",deckString);
+  }
+
   return (
-    <div className="container text-center">
-      <div className="row">
-        <div className="text-center p-5">
-          <h1>CreateDeck</h1>
+    <>
+      {/* DeckPicker */}
+      <div className="container" hidden={hiddenDeck}>
+        <div className="row">
+          <div className="col-2 p-3"><h1>Decks</h1></div>
+          <div className="col-7"></div>
+          <div className="col-3 d-grid gap-2 p-3">
+            <button type="button" className="btn btn-primary" onClick={()=>{navigateTo("/");}}>Back</button>
+          </div>
+        </div>
+        <div className="row p-4"></div>
+        <div className="row">
+          <button type="button" className="btn btn-primary" onClick={()=>{addDeck()}}>Create New Deck</button>
+        </div>
+        <div className="row p-3">
```

Comment:
```suggestion
        <div className="row py-3">
```
Det er jo kun horizontal padding i vil have, derfor kan i bruge py-3 istedet for p-3.